src/mongocxx/private/ssl.hpp

```diff
@@ -26,20 +26,21 @@ namespace options {
 #if defined(MONGOC_HAVE_SSL)
 inline ::mongoc_ssl_opt_t make_ssl_opts(const ssl& ssl_opts) {
     ::mongoc_ssl_opt_t out;
+    memset(&out, 0, sizeof(out));
```

Comment:
I think you can just write this as:

::mongoc_ssl_opt_t out{};

Instead of needing to call memset.